src/app/tests/suites/certification/ci-pics-values

```diff
@@ -922,6 +922,9 @@ MCORE.BDX.Sender=0
 MCORE.BDX.SynchronousReceiver=0
 MCORE.BDX.SynchronousSender=0
 
+# Fabric Synchronization
+MCORE.FS=1
```

Comment:
This enable the FS feature by default in SDK, we have passed the SVE certs, but need the certification team to approve this feature in the spec

Reply:
I just built the ToT for the Matter spec and noticed that the FS is already included in the ToT. I believe we can enable this feature now and don’t need to wait for the official 1.4 spec. I'll submit another PR to remove the provisional status of the XML.